src/phoenix/server/encryption.py

```diff
@@ -0,0 +1,136 @@
+"""Encryption utilities for sensitive data storage using Fernet.
+
+Security Model:
+---------------
+This module provides encryption-at-rest for sensitive configuration data and secrets
+stored in the Phoenix database. The encryption uses Fernet (symmetric encryption based
+on AES-128-CBC with HMAC authentication).
+
+Key Derivation:
+- The encryption key is derived from the Phoenix server's secret using PBKDF2-HMAC-SHA256
+- A fixed public salt is used (see _ENCRYPTION_KEY_DERIVATION_SALT below)
+- The salt doesn't need to be secret; its purpose is to prevent rainbow table attacks
+  against the password/secret, and to ensure different installations derive different keys
+- 100,000 iterations are used (OWASP recommended minimum as of 2023)
+
+Threat Model:
+- Protects against: Database dumps falling into unauthorized hands
+- Does NOT protect against: Attackers who have access to both the database AND the server secret
+- Key rotation: Changing the server secret will make existing encrypted data unreadable.
+  There is currently no key rotation mechanism - this is a known limitation.
```

Comment:
I'm not sure I'd leave this in. It's gonna be triggering to consumers of this code.